tartufo/scanner.py

```diff
@@ -336,6 +339,36 @@ def should_scan(self, file_path: str):
             return False
         return True
 
+    @property
+    def config_data(self):
+        return self._config_data
+
+    @config_data.setter
+    def config_data(self, data) -> MutableMapping[str, Any]:
+        self._config_data = data
+        return self._config_data
```

Comment:
```suggestion
    def config_data(self, data: MutableMapping[str, Any]) -> None:
        self._config_data = data
```
Returning data from a setter in Python is generally not a valuable operation, because there's no way to access it. You would need something like...

```python
new_data = scanner.config_data = data
```
Which is just confusing. And probably won't even work the way you expect.

You can find a bit more of an explanation here, if you're interested: https://stackoverflow.com/a/16615910